src/components/pages/AccountSettings/RenderAccountSettings.js

```diff
@@ -116,6 +116,10 @@ function RenderAccountSettings() {
         style={unlock ? { opacity: '.3' } : null}
       >
         <AccountSettingsForm disabled={unlock} />
+        <div classname="settings-buttons-cotainer">
+          <Button disabled={unlock}>Edit Credit Card Info</Button>
+          <Button disabled={unlock}>Edit Subscription Plan</Button>
+        </div>
       </div>
     </div>
   );
```

Comment:
Nice work - if you look in the less for the accountSettings page you should be able to position these pretty easily using the css grid that is overlaid over the entire component - let me know if you haven't used grid before and I can run through it with you - super simple, very semantic also so you should be able to see how it works just by looking at what rules are applied to the other button.